plugin/trino-hive/src/test/java/io/trino/plugin/hive/metastore/cache/TestCachingHiveMetastoreWithQueryRunner.java

```diff
@@ -51,35 +53,34 @@
     private static final String ALICE_NAME = "alice";
     private static final Session ALICE = getTestSession(new Identity.Builder(ALICE_NAME).build());
 
-    private DistributedQueryRunner queryRunner;
-    private File temporaryMetastoreDirectory;
+    private static final int nodeCount = 3;
 
-    @BeforeMethod
-    public void createQueryRunner()
+    private FileHiveMetastore fileHiveMetastore;
+
+    @Override
+    protected QueryRunner createQueryRunner()
             throws Exception
     {
-        queryRunner = DistributedQueryRunner
+        DistributedQueryRunner queryRunner = DistributedQueryRunner
```

Comment:
Why not use `HiveQueryRunner`?

Reply:
It required adoption in HiveQueryRunner. As I wanted to minimise changes I left this part unchanged.
But as it already made to more that single file change, let me just move to HiveQueryRunner.

Reply:
Done